packages/edit-site/src/components/posts-app/posts-list.js

```diff
@@ -395,6 +396,11 @@ export default function PostsList( { postType } ) {
 						value: id,
 						label: name,
 					} ) ) || [],
+				render: ( { item } ) => {
+					return (
```

Comment:
I don't think we should reuse this render field function. Posts list is meant to be reused by every post type eventually, have an API to get the fields by `postType`, but I see templates a bit different than the 'normal' post types. Maybe I'm wrong, but it wouldn't hurt to duplicate just some code for now and not use `useAddedBy` hook for pages/posts. Is there a case where non actual users create pages/posts as authors? 🤔

Reply:
Hi @ntsekouras, I applied your suggestion and did not reuse the field from the templates.